hazelcast/src/main/java/com/hazelcast/map/impl/EntryViews.java

```diff
@@ -44,4 +47,35 @@ private EntryViews() {
                 .withExpirationTime(record.getExpirationTime())
                 .withLastStoredTime(record.getLastStoredTime());
     }
+
+    public static <K, V> WanMapEntryView<K, V> createWanEntryView(Data key, Data value,
+                                                                  Record<V> record,
+                                                                  SerializationService serializationService) {
+        return new WanMapEntryView<K, V>(key, value, serializationService)
+                .withCost(record.getCost())
+                .withVersion(record.getVersion())
+                .withHits(record.getHits())
+                .withLastAccessTime(record.getLastAccessTime())
+                .withLastUpdateTime(record.getLastUpdateTime())
+                .withTtl(record.getTtl())
+                .withMaxIdle(record.getMaxIdle())
+                .withCreationTime(record.getCreationTime())
+                .withExpirationTime(record.getExpirationTime())
+                .withLastStoredTime(record.getLastStoredTime());
+    }
+
+    public static <K, V> EntryView<K, V> toLazyEntryView(EntryView<K, V> entryView,
```

Comment:
not used?

Reply:
Yes, it's not used. Not sure if I should remove it though, maybe someone will need a lazy deserializing entry view in the future?

Reply:
maybe it can be added back when necessity appears. I vote for deletion.

Reply:
OK, will remove.